src/components/chips/BuildStatusChip.tsx

```diff
@@ -5,12 +5,12 @@ import Tooltip from '@material-ui/core/Tooltip';
 import { graphql } from 'babel-plugin-relay/macro';
 import React from 'react';
 import { createFragmentContainer, Disposable, requestSubscription } from 'react-relay';
-import { cirrusColors } from '../../cirrusTheme';
 import environment from '../../createRelayEnvironment';
 import { buildStatusColor } from '../../utils/colors';
 import { buildStatusIconName, buildStatusMessage, isBuildFinalStatus } from '../../utils/status';
 import { formatDuration } from '../../utils/time';
 import { BuildStatusChip_build } from './__generated__/BuildStatusChip_build.graphql';
+import { WithTheme, withTheme } from '@material-ui/core';
```

Comment:
Ideally we should avoid this to speed up the development server but I suppose its fine for now